### ISSUE TITLE
Accessibility Bug  : Narrator does not read the sorting option for column header i.e. "Aria-sort".

### ISSUE THREAD

**Issue by vinapawar17** [microsoft/fluentui#16657]
Trying to read Aria-sort property for detail Grid but Narrator is not supporting to read any column header which is clicked/sorted.
Please confirm whether Aria-sort is supported by Narrator + Edge/chrome or Not.

Describe the issue:

There's grid with sorting button beside column header.
on click of sorting button it should read ascending/descending/not sorted 

Actual behavior:

at column headers screen reader does not announce status regarding that the data is sorted in ascending or descending order.

Expected behavior:

at column headers screen reader should announce status regarding that the data is sorted in ascending or descending order.

**Comment by gouttierre** [comment]
@vinapawar17 - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the COMPONENT examples (click “Export to CodePen”) or https://aka.ms/fluentpen.. In order for us to address the ARIA spec correctly - we will need to run the Accessibility Insights tool on your CodePen that is using our components to verify errors and functionality.
